### DISCUSSION TITLE
How can I find the directories latest version?

### DISCUSSION THREAD

**Discussion by blueray453** [borgbackup/borg#7205] [Q&A] [answered]
I have a folder named `_Production`. I deleted it few days back. It is somewhere in the older archives. How can I find the directories latest version.

I used `borg mount -o versions . /tmp/mymountpoint`. I can see the folder. But not sure which archive it belongs to. 

What can I do?

**Comment by ThomasWaldmann** [comment, marked as answer]
The versions view manages versions for the leaf objects of the directories/files tree which are **files**.

Guess it is not the tool you need. But you could use the normal view of `borg mount` and just `ls -ld /tmp/mountpoint/*/directorywanted` (or so) to find the most recent archive that has this directory.
